app/websocket/websocketclient.c

```diff
@@ -509,7 +539,7 @@ static void ws_initReceiveCallback(void *arg, char *buf, unsigned short len) {
   }
 
   // Check server has valid sec key
-  if (strstr(buf, WS_HTTP_SEC_WEBSOCKET_ACCEPT) == NULL || strstr(buf, ws->expectedSecKey) == NULL) {
```

Comment:
Hmm, I've removed this because checking presence of WS_HTTP_SEC_WEBSOCKET_ACCEPT was case-sensitive (for example it not worked for Amazon's MQTT websocket gateway which sends this header key lowercase). We still check if this header value (SHA-1 hash) is valid, so it is not big deal I think.
To do this properly we probably should parse all response headers and store them in header_t structs, instead of using this strstr search.